release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -210,6 +210,35 @@ For more information, see xref:../post_installation_configuration/cluster-capabi
 
 * With this update, in the *Add+* page of the developer perspective, a new menu added to share project and Helm Chart repositories that allows to add or remove users to the project.
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic plug-in updates
+
+With this update, you can use the new `console.openshift.io/default-i18next-namespace` annotation to determine if the `ConsolePlugin` contains localization resources. If the annotation is set to `"true"`, the localization resources from the i18n namespace named after the dynamic plug-in, are loaded. If the annotation is set to any other value or is missing on the `ConsolePlugin` resource, localization resources are not loaded.
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dynamic-plug-ins_web_console[Dynamic plug-ins].
+
+[id="ocp-4-11-dark-mode"]
+==== Support for dark mode theme
+
+The {product-title} web console now supports the dark mode theme. On the *User Preferences* page, select your preferred theme to view the web console in.
+
+[id="ocp-globally-installed-operators-display-operand-instances"]
+==== Display operand instances for all managed namespaces on the *Installed Operator* page
+
+With this update, the *Operator* -> *Installed Operator* page will show all Operators across all namespaces. You are still able to view only the instances in the selected namespace within the project selector. When viewing the operand instances, a new switching control allows all operand instances from either all namespaces or only the current namespace to be seen.
+
+[id="ocp-4-11-conditional-updates"]
+==== Conditional updates
+
+With this update, if conditional updates are available, you can enable `Include supported but not recommended versions` in the `Select new version` dropdown of the *Update cluster* modal to populate the dropdown list with conditional updates. If a `Supported but not recommended` version is selected, an alert will appear below the dropdown displaying potential issues with the version.
+
+[id="ocp-4-11-pod-disruption-budgets"]
+==== Pod disruption budgets (PDBs)
+
+This update provides support for pod disruption budgets (PDBs) to the {product-title} web console. From *Workloads* -> *PodDisruptionBudgets*, you can create PDBs for pod resources. You can select `maxUnavailable` and `minAvailable` from the availability requirement list and set the value of pods running. Alternatively, pod disruption budgets can be created from `pod controller resources` list and *Detail* pages. For example, from *Workloads* -> *Deployments* click `Add PodDisruptionBudget`.
```

Comment:
should it be "requirements list"?

Reply:
Its in the UI as `availability requirement`